cocos/platform/android/jni/JniHelper.h

```diff
@@ -85,6 +90,10 @@ class CC_DLL JniHelper
         }
     }
 
+    /**
+    @brief Check whether there are proper JniMethodInfo
+    @return true if there are proper JniMethodInfo; otherwise false & report Error.
```

Comment:
it will return value from Java static boolean method. If no such method then will return false

Reply:
thanks.